core/tools/compileCSSModules.js

```diff
@@ -64,8 +60,9 @@ const modules = [
 
 const compileCSSModules = () => {
   modules.forEach((module, idx) => {
-    const outFile = path.resolve(__dirname, `../css/components/${module}.css`);
-    const mapFile = path.resolve(__dirname, `../css/components/${module}.map.css`);
+    const componentsDirectory = path.resolve(__dirname, `../css/components/${module}`);
```

Comment:
@bfbiggs will this affect other things? The structure changes here

Reply:
Correct, it enables the components to each have their own folders. Previously it was resolving all CSS  and map files inside one components directory, less organized.

Reply:
Correct but you need to verify with @pauljeter if this affects what is output on the cdn.

Reply:
Copy that. I could use either solution ultimately, but thought this was a logical structure. What do you think @pauljeter ?

Reply:
Not sure how it would be better organized? Instead of flat component css files, there would be a folder for each component with only 1 file. I'm good either way. The path will be a little redundant with the folder structure. 
`css/components/button/button.css` vs `css/components/button.css

Reply:
Also, @bfbiggs, we are not currently publishing the components css to NPM or the CDN. This code was here from when I was exploring the option of publishing the modules.

Reply:
I'm re-working this bit to skip the map files and compile to a flat `css/components` directory